templates/internet-of-things/smart-city.html

```diff
@@ -37,7 +37,7 @@ <h1>Building the future with smart cities</h1>
         <div class="p-logo-section__item">
           {{ image (
             url="https://assets.ubuntu.com/v1/e99b843a-Bosch_Logo-flat.svg",
-            alt="",
+            alt="Bosch logo",
```

Comment:
Assistive tech will read out: 
"Trusted by leading service providers, globally. Bosch logo. Tesla logo" etc etc

In this context, it would be a better experience to have them read out:
"Trusted by leading service providers, globally. Bosch. Tesla." etc etc

```suggestion
            alt="Bosch",
```